roles/keycloak_quarkus/templates/keycloak.conf.j2

```diff
@@ -10,6 +10,10 @@ db-password={{ keycloak_quarkus_db_pass }}
 {% endif %}
 {% endif %}
 
+{% if keycloak_quarkus_hostname_strict_https -%}
+hostname-strict-https={{ keycloak_quarkus_hostname_strict_https }}
```

Comment:
you need `| lower` or python will emit False instead of false, which quarkus doesn't like

Reply:
plus, please add the role parameter to meta/argument_specs.yml, with `required: no` and telling in the description that it only applies when `keycloak_quarkus_frontend_url` is not set